scripts/run_simulator_project.py

```
@@ -0,0 +1,38 @@
+Import("env")  # noqa: F821 - SCons injects this at build time
+import builtins
+
+RUN_SIMULATOR_TARGET_KEY = "_crosspoint_run_simulator_target_registered"
+
+
+def run_simulator(source, target, env):
+    import os
+    import subprocess
+
+    binary = env.subst("$BUILD_DIR/program")
+    result = subprocess.run([binary], cwd=os.getcwd())
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)